### DISCUSSION TITLE
User Search function

### DISCUSSION THREAD

**Discussion by StevenForler** [night/betterttv#6673] [Ideas]
I'm starting to watch twitch more and more and I realized I've been under utilizing BTTV. I thought it would be a great Idea to find the users of BTTV of streamers that you follow so that you know which emotes they use in their chat and maybe discord so I don't have to keep guessing which one is shared  in their specific.

If this function exists, I'm just not aware of its location or how to do it. Please let me know if this is already a thing and I'm just brain dead and needed to look harder

**Comment by night** [comment]
Closing this one out — to see which emotes are available in any chat, open BetterTTV's emote menu (the smiley icon next to chat input). It groups emotes by channel and global sets so you can see what's available in the channel you're currently in. If you wanted something different, please open a new discussion with more detail.
